src/client/Components/ShowCollection/ShowCollection.js

```diff
@@ -0,0 +1,50 @@
+import styles from './ShowCollection.module.css';
```

Comment:
This will need to be updated to follow in the consistency of the rest of the files.